### PR TITLE
Updates Development to iPhone 11 Sim

### PR DESCRIPTION
**Pull request by redreceipt**
## DESCRIPTION

Screen Shot 2019-09-22 at 2 33 24 PM

### What does this PR do, or why is it needed?

This updates our `yarn ios` sim to iPhone 11 for two reasons:

1. iPhone X disappeared from the list when XCode updated and by build failed.
2. We should stay current with the latest. It's a HUGE screen.

### What design trade-offs/decisions were made?

None.

### How do I test this PR?

Make sure Xcode is the latest version.
`yarn ios`

### What automated tests did you write?

None.

## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [x] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [x] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apolloschurchapp/package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "android": "SIM=\"$($HOME/Library/Android/sdk/emulator/emulator -list-avds | sed -n 1p)\" && $HOME/Library/Android/sdk/emulator/emulator -avd \"${SIM}\" & react-native run-android --appId com.apolloschurch.app",
     "codecov": "cat ./coverage/lcov.info | codecov",
     "fixlint": "eslint ./src --fix",
-    "ios": "react-native run-ios --simulator=\"iPhone X\"",
+    "ios": "react-native run-ios --simulator=\"iPhone 11\"",
```
Comment (vinnyjth) [inline packages/apolloschurchapp/package.json]: Can we take out the `--simulator` part all together? I'm worried that people who haven't updated Xcode yet will end up the opposite problem :)

**Comment by redreceipt** [comment]
sadly no this is error

Screen Shot 2019-09-23 at 1 02 51 PM

seems it looks for iPhone X by default. I just didn't figure it was stretch if we're people to stay up to date with cocoapods on their own, is Xcode a big ask?

**Comment by vinnyjth** [comment]
Bummer! okay this PR is great. I and Burke are already upgraded and we didn't even try :)

**Comment by redreceipt** [comment]
gimme that thumbs up, the "merge" label isn't enough 😉

**Pull request merged**